docs/libtorch.rst

```diff
@@ -34,3 +34,18 @@ To produce libtorch.a rather than libtorch.so, set the environment variable `BUI
 To use ninja rather than make, set `CMAKE_GENERATOR="-GNinja" CMAKE_INSTALL="ninja install"`.
 
 Note that we are working on eliminating tools/build_pytorch_libs.sh in favor of a unified cmake build.
+
+Building libtorch without using Python
```

Comment:
ATen, which is part of libtorch,  can not be built from source  if Python3 runtime + dataclasses + pyyaml are missing on the system, see https://github.com/pytorch/pytorch/blob/24df3b7373a80fbccebfde72f70fdd045ec44430/cmake/Codegen.cmake#L185
```suggestion
Building libtorch using CMake
```